client/blocks/credit-card/use-element-options.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect, useCallback } from '@wordpress/element';
+
+/**
+ * @typedef {import('../stripe-utils/type-defs').StripeElementOptions} StripeElementOptions
+ */
+
+/**
+ * Returns the value of a specific CSS property for the element matched by the provided selector.
+ *
+ * @param {string} selector     CSS selector that matches the element to query.
+ * @param {string} property     Name of the property to retrieve the style
+ *                              value from.
+ * @param {string} defaultValue Fallback value if the value for the property
+ *                              could not be retrieved.
+ *
+ * @return {string} The style value of that property in the document element.
+ */
+const getComputedStyle = ( selector, property, defaultValue ) => {
+	let elementStyle = {};
+
+	if (
+		typeof document === 'object' &&
+		typeof document.querySelector === 'function' &&
+		typeof window.getComputedStyle === 'function'
```

Comment:
Let's use the established [yoda condition WP standard](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions) even though that's _technically_ only defined for PHP 🙂

Reply:
- GB doesn't use Yoda,
- `@woocommerce/eslint-plugin` doesn't use yoda.
- Woo Blocks doesn't use yoda (by virtue of the `@woocommerce/eslint-plugin`). Why use yoda?

Reply:
> Why use yoda?

To follow WP coding standards, and to follow already established conventions throughout the codebase (even if yoda conditions are only strictly required in PHP).

> GB doesn't use Yoda, `@woocommerce/eslint-plugin` doesn't use yoda, Woo Blocks doesn't use yoda

I wasn't aware of any of this! It's interesting for sure.

This isn't a hill I'm willing to die on, but it caught my eye as something that stands out compared to the rest of the codebase. We'll discuss it further :slightly_smiling_face:

Reply:
We will not be using yoda conditions as per discussion in pc4etw-8L-p2#comment-469. Marking this as resolved.